src/molecules/RegistrationForm/index.js

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import {useState, useMemo, useCallback, memo} from "react";
+import './style.scss';
+import Typography from "../../atoms/Typography";
+import Button from "../../atoms/Button";
+import {Link, useNavigate} from "react-router-dom";
+import axios from 'axios';
+import InputLabel from "../../atoms/FormInputLabel";
+import WSLogo from "../../atoms/icons/WS_logo.png";
+import LogoMain from "../../atoms/icons/LOGOMAIN.png";
+
+const initialFormData = {
+    username: '',
+    email: '',
+    password: '',
+    acceptTC: false,
+};
+
+function RegistrationForm() {
+    const [formData, setFormData] = useState(initialFormData);
+    const [response, setResponse] = useState(null);
+    const [error, setError] = useState(null);
+    const navigate = useNavigate();
+
+    const handleChange = (event) => {
+        const { name, value } = event.target;
+        setFormData({
+            ...formData,
+            [name]: value,
+        });
+    }
+
+    const handleSubmit = useCallback((event) => {
+        event.preventDefault();
+        console.log(formData)
+        if (!formData.username || !formData.email || !formData.password || !formData.acceptTC) {
+            setError('Please fill in all fields');
+            return;
+        }
+        axios.post('/api/v1/auth/register', formData).then((response) => {
+            if (response.status === 201) {
+                localStorage.setItem('username', response.data.jwtToken);
```

Comment:
I think it should be vice versa:
```suggestion
                localStorage.setItem('username', response.data.username);
                localStorage.setItem('jwtToken', response.data.jwtToken);
```